Lib/test/test_type_comments.py

```diff
@@ -155,80 +175,113 @@ def favk(
 
 class TypeCommentTests(unittest.TestCase):
 
-    def parse(self, source):
-        return ast.parse(source, type_comments=True)
+    lowest = 4  # Lowest minor version supported
+    highest = sys.version_info[1]  # Highest minor version
+
+    def parse(self, source, feature_version=highest):
+        return ast.parse(source, type_comments=True,
+                         feature_version=feature_version)
+
+    def parses(self, source, minver=lowest, maxver=highest, expected_regex=""):
```

Comment:
`parses` is not very descriptive and is easy to confuse with `parse`, maybe `parse_all`, or `parse_all_versions`?